### PR TITLE
PoC improve component options

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
#### Summary of Changes

I think the screen shoots speak for themself

image

image

image

image

#### Testing Instructions

Apply patch and test config options.

### CODE REVIEW

File: administrator/components/com_patchtester/config.xml [hunk 1]
```diff
@@ -21,29 +37,55 @@
...
 			description="COM_PATCHTESTER_FIELD_REPO_DESC"
 			default="joomla-cms"
+			id="repo"
+			showon="repository:custom"
 		/>
 
+	</fieldset>
+
+	<fieldset
+		name="component"
```
Comment (mbabker) [inline administrator/components/com_patchtester/config.xml]: You're using the same fieldset name here as the above fieldset.

Comment (andrepereiradasilva) [inline administrator/components/com_patchtester/config.xml]: ups

Comment (andrepereiradasilva) [inline administrator/components/com_patchtester/config.xml]: solved.

BTW do you know any way to solve the autocomplete

The only way i know is to change the field type from password to something else.

Comment (mbabker) [inline administrator/components/com_patchtester/config.xml]: It's been a reoccurring issue for a while (I think there are core hacks in place to deal with Firefox having the same misbehavings).

Comment (andrepereiradasilva) [inline administrator/components/com_patchtester/config.xml]: i notice, for instance, the ftp password field doesn't auto popuplate, don't know why yet.

will check

**Comment by andrepereiradasilva** [comment]
ok solved the autocomplete.
it seems we just need to a white space as default in the username

**Pull request merged**